fix: fix type errors caused by unsuccessful queries

EdieLemoine opened the pull request.

EdieLemoine committed:
fix: fix type errors caused by unsuccessful queries

joerivanveen commented on src/Factory/Consignment/ConsignmentFactory.php:
```diff
@@ -207,18 +206,18 @@ private function getFormattedLabelDescription(): string
         $this->orderObject       = $order;
-        $this->orderData         = OrderLabel::getDataForLabelsCreate([$order->getId()])[0];
+        $orderData               = OrderLabel::getDataForLabelsCreate([$order->getId()])[0];
+        $this->orderData         = $orderData;
```
```suggestion
        $this->orderData         = OrderLabel::getDataForLabelsCreate([$order->getId()])[0];
```

EdieLemoine committed:
Update src/Factory/Consignment/ConsignmentFactory.php

Co-authored-by: Joeri van Veen <joeri@myparcel.nl>

EdieLemoine committed:
fix undefined index in consignmentfactory

EdieLemoine committed:
reduce type errors in order list

EdieLemoine commented:
/rebase

joerivanveen commented on src/Factory/Consignment/ConsignmentFactory.php:
```diff
@@ -194,10 +194,7 @@ private function getDeliveryType(): int
+        $labelDescription = $this->getLabelDescription();
 
         if (strlen($labelDescription) > Constant::ORDER_DESCRIPTION_MAX_LENGTH) {
             return substr($labelDescription, 0, 42) . '...';
```
```suggestion
            return substr($labelDescription, 0, Constant::ORDER_DESCRIPTION_MAX_LENGTH - 3) . '...';
```

EdieLemoine committed:
Update src/Factory/Consignment/ConsignmentFactory.php

Co-authored-by: Joeri van Veen <joeri@myparcel.nl>

Commit:
chore(release): 1.5.3 [skip ci]

### [1.5.3](https://github.com/myparcelnl/prestashop/compare/v1.5.2...v1.5.3) (2022-04-19)

### :bug: Bug Fixes

* error when performing search with no results ([#139](https://github.com/myparcelnl/prestashop/issues/139)) ([e7ac29d](https://github.com/myparcelnl/prestashop/commit/e7ac29db1a599db3c7ede8fb7befdb7be09dc9eb))
* fatal error due to nonexistent listener ([#141](https://github.com/myparcelnl/prestashop/issues/141)) ([7aca74d](https://github.com/myparcelnl/prestashop/commit/7aca74da87fd92224a99953903bb27734a693b90))
* fix insurance settings ([#128](https://github.com/myparcelnl/prestashop/issues/128)) ([0de46ae](https://github.com/myparcelnl/prestashop/commit/0de46ae54dee13702f51aeacaf342a609ad2193f))
* only send status change email if status actually changed ([#124](https://github.com/myparcelnl/prestashop/issues/124)) ([cb06ff1](https://github.com/myparcelnl/prestashop/commit/cb06ff1f2453e671503f3a3cf0681bc855ea29b5))
* resolve scripts to paths based on site url ([#134](https://github.com/myparcelnl/prestashop/issues/134)) ([a102ea3](https://github.com/myparcelnl/prestashop/commit/a102ea3771db43b46aae60345a6afe2a88b6bfa5))

EdieLemoine committed:
fix: errors implementing Throwable not caught
